src/oas2/__tests__/accessors.test.ts

```diff
@@ -20,13 +22,90 @@ const securityDefinitionsFixture: Dictionary<Security> = {
 };
 
 describe('accessors', () => {
-  const securityFixture = [
+  const securityFixture: Security[] = [
```

Comment:
Maybe it's not really worth to rename the security definition on top and create a new alias just because of this fixture.